src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +344,40 @@ Demos
 
 ## FAQ
 
+**Do I need the `--disable-gpu` flag?**
```

Comment:
I've just removed that other one for now. Easier just to remove this whole entry when chrome 60 comes out.